pyvips/voperation.py

```diff
@@ -22,6 +22,124 @@
 _OPERATION_DEPRECATED = 8
 
 
+class Introspect(object):
+    """Build introspection data for operations.
+
+    Make an operation, introspect it, and build a structure representing
+    everything we know about it.
+
+    """
+
+    def __init__(self, operation_name):
+        op = Operation.new_from_name(operation_name)
```

Comment:
We're constructing a new operation now twice, see:
https://github.com/libvips/pyvips/blob/5f7c5d3c823497d494141e0f900562824ed12edb/pyvips/voperation.py#L220-L228

Perhaps we should only do this once?

Reply:
The one in `Introspect` is only made once (after that, intro data is fetched from the cache), so I don't think it matters.